### ISSUE TITLE
Can we change the size of plots? Tried figsize parameter but got an error. 

### ISSUE THREAD

**Issue by soliverc** [facebook/prophet#1182]
I searched the issues here for references to changing plot size and saw this suggestion which seems to have been implemented #969 

However it doesn't seem to be working. Was it removed?

Or am I doing something wrong? (probably this)

What I tried:

```python
p = m.plot(forecast,figsize=(15,10))

TypeError                                 Traceback (most recent call last)
<ipython-input-59-e9e6f5b629c6> in <module>
      4 forecast = m.predict(future)
      5 
----> 6 p = m.plot(forecast,figsize=(10, 6))

TypeError: plot() got an unexpected keyword argument 'figsize'
```

**Comment by bletham** [comment]
#969 landed right after the v0.5 release, and we haven't yet pushed the next version to pypi that will include that.

In the meantime, `m.plot` just calls `fbprophet.plot.plot`, which does in v0.5 have the figsize argument. So you can do
```
from fbprophet.plot import plot
plot(m, fcst, figsize=(8, 4))
```
(see also #1107)

**Comment by soliverc** [comment]
Good to know, thanks.
